src/main/kotlin/no/nav/syfo/sykmelder/api/SykmelderApi.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.syfo.sykmelder.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.route
+import java.util.UUID
+import no.nav.syfo.log
+import no.nav.syfo.sykmelder.service.SykmelderService
+import no.nav.syfo.util.getAccessTokenFromAuthHeader
+
+fun Route.sykmelderApi(
```

Comment:
Skal det å hente sykmelder være et eget api? Skal frontend gi mulighet til å trykke på en knapp for å hente sykmelder, eller noe slikt..? Tror ikke jeg helt skjønner endringen dere har gjort 😅

Reply:
Jepp! Stemmer :) Men man henter uten å trykke på en knapp.